### DISCUSSION TITLE
Create Player from UUID (Or should I?)

### DISCUSSION THREAD

**Discussion by zax71** [Minestom/Minestom#1842] [Help] [answered]
Hi! I'm creating a Class to read from an SQLite database that stores a player's UUID. How should I return this player, as a `String` of `MojangUtils.fromUuid(uuid)` or somehow creating a `Player` object of that UUID. To create a player object I seam to need a `playerConnection` but I'm not sure how to make one of those.

**Comment by KrystilizeNevaDies** [comment]
What are you trying to do? You can't really create a player in that way.
\

**Reply by zax71** [reply]
Maybe I've got the wrong end of the stick, I get the player from the database (as a UUID) and I'm not sure what I should return

**Reply by KrystilizeNevaDies** [reply]
You should return the UUID? Are you trying to find what the player's name is?

**Reply by zax71** [reply]
I guess the player name is probably the most useful thing to return for leaderboards etc

**Reply by KrystilizeNevaDies** [reply]
There should be a method in MojangUtils for that.

**Reply by bebranator** [reply]
idk but "connectionManager.getPlayer(ur_uuid)"

**Reply by zax71** [reply, marked as answer]
> There should be a method in MojangUtils for that.

Ah, I was put off by that returning a jsonArray before, but I can just use `.get("name")`
